CONTRIBUTING.md

```diff
@@ -35,15 +36,23 @@ that the description is in Markdown, so reformat from Rst if necessary).
 Tag the release according to [semantic versioning
 guidelines](https://semver.org/), preceded with a `v` (e.g.: v1.0.0). The
 release title is the tag and the release date together (e.g.: v1.0.0
-(2019-07-25)). Tick the pre-release box in case the release is a candidate
-release, and amend the version tag with `rc` and the candidate number.
+(2019-07-25)).
+
+### Release candidate
+When releasing a release candidate on Github, tick the pre-release box, and
+amend the version tag with `rc` and the candidate number. Ensure the release
+candidate version is accurate in `CITATION.cff` and `era5cli/__version__.py`.
+If the version number in these files is not updated, Zenodo and RSD release
```

Comment:
It not working on RSD is an effect of the failed Zenodo entry. As RSD just fetches the list of versioned Zenodo records belonging to a concept DOI. I would not mention the RSD.
```suggestion
If the version number in these files is not updated, Zenodo release
```